ccd-config-generator/src/main/java/uk/gov/hmcts/ccd/sdk/CaseEventGenerator.java

```diff
@@ -68,25 +69,30 @@ public void writeEvents(File root, ResolvedCCDConfig<T, S, R> config) {
     data.put("PostConditionState", toCCDStateString(event.getPostState(), allStates));
     data.put("SecurityClassification", "Public");
 
-    if (event.getAboutToStartURL() != null) {
-      data.put("CallBackURLAboutToStartEvent", event.getAboutToStartURL());
+    if (event.getAboutToStartCallback() != null) {
+      String url = callbackHost + "/callbacks/about-to-start";
```

Comment:
it would be useful to expose "/callbacks/about-to-start" as a constant for the functional tests